contents/handbook/company/team/charles-cook.md

```diff
@@ -9,13 +9,12 @@ This guide might be helpful in working with me.
 ## Areas of responsibility
 
 - Making sure all our business operations run smoothly
-- Overseeing all things marketing, including our external paid ads agency
-- All of our finance stuff (accounting, tax etc.)
+- Overseeing all things marketing, including managing our external paid ads agency
+- All of our financial reporting and planning, including managing our external accountants
 - Any legal matters, including compliance and privacy - I am the main point of contact with our lawyers
-- Sales operations, ie. making sure we follow up with customers, generating quotes
-- Making sure our customer support systems run smoothly
+- Making sure our customer support systems (ie. Papercups) run smoothly
```

Comment:
This could become @camerondeleone (depending where you go to deciding where to focus)